File: backend/java-springboot/real-estate-backend/src/main/java/com/demo/real_estate_backend/model/ViewingStatus.java
package com.demo.real_estate_backend.model;

public enum ViewingStatus {
    PENDING,      // Viewing request submitted, awaiting confirmation
    CONFIRMED,    // Viewing confirmed by agent/owner
    REJECTED,     // Viewing request rejected
    COMPLETED,    // Viewing was completed
    CANCELLED     // Viewing was cancelled by user
}

File: backend/real_estate_dotnet/Models/Enums/ViewingStatus.cs
using real_estate_dotnet.Models.Enums;
using real_estate_dotnet.Models;


namespace real_estate_dotnet.Models
{
    public enum ViewingStatus
    {
        Pending,     // Viewing request submitted, awaiting confirmation
        Confirmed,   // Viewing confirmed by agent/owner
        Rejected,    // Viewing request rejected
        Completed,   // Viewing was completed
        Cancelled    // Viewing was cancelled by user
    }
}
